govcd/vdc.go

```diff
@@ -353,10 +352,10 @@ func (vdc *Vdc) FindEdgeGateway(edgegateway string) (EdgeGateway, error) {
 
 }
 
-// getEdgeGatewayByHref retrieves an edge gateway from VDC
+// GetEdgeGatewayByHref retrieves an edge gateway from VDC
 // by querying directly its HREF.
 // The name passed as parameter is only used for error reporting
-func (vdc *Vdc) getEdgeGatewayByHref(name, href string) (*EdgeGateway, error) {
+func (vdc *Vdc) GetEdgeGatewayByHref(name, href string) (*EdgeGateway, error) {
```

Comment:
IMHO name is extraneous here. There may be cases when one doesn't have name and it would be some random string

Reply:
Thanks. Removed